### PR TITLE
chore: updated extension-tao-mediamanager to 12.20.4

### PR DESCRIPTION
**Pull request by pnal**
Related ticket: https://oat-sa.atlassian.net/browse/AUT-1620

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -50,7 +50,7 @@
     "oat-sa/extension-tao-lti": "15.5.0",
     "oat-sa/extension-tao-ltideliveryprovider": "12.9.0",
     "oat-sa/extension-tao-revision": "10.2.0",
-    "oat-sa/extension-tao-mediamanager": "12.20.3",
+    "oat-sa/extension-tao-mediamanager": "12.21.1",
```
Comment (bziondik) [inline composer.json:53]: Why 12.20.3 to 12.21.1? should be 12.20.3 to 12.20.4 ?

Comment (pnal) [reply composer.json:53]: 12.20.3 and 12.20.3 have only one difference: https://github.com/oat-sa/extension-tao-mediamanager/pull/359/files
all is about tests and does not affect business logic, so for me it does not make sense to cherry-pick and release one more minor version.
wdyt?

Comment (pnal) [reply composer.json:53]: I will update this PR

**Review by andreluizmachado — APPROVED** [review]
LGTM

**Pull request merged**